### PR TITLE
Studio GUI preview documentation section

### LINKED ISSUE #494: Add the Taipy Studio GUI preview section
**Description**

Taipy Studio GUI leverages the Markdown preview feature of Visual Studio Code. 
We want that thoroughly documented.

**Expected change**

The Taipy Studio section has a specific section on the matter, including mock data management.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #494.

Fixed setup_generation launch documentation

### CODE REVIEW

File: docs/manuals/studio/gui.md [hunk 1]
```diff
@@ -72,6 +72,147 @@ set to the property is proposed to the user:
   <img src="../images/autocomplete_function.gif" width="75%"/>
 </p>
 
+## Page preview
+
+If you are used to working with Markdown in Visual Studio Code, you certainly came across the
+[Markdown preview](https://code.visualstudio.com/docs/languages/markdown#_markdown-preview)
+feature: pressing `Ctrl+Shift+V` in an editor that contains Markdown content creates a window
```
Comment (dinhlongviolin1) [inline docs/manuals/studio/gui.md]: Might be different depending on user os

File: docs/manuals/studio/gui.md [hunk 2]
```diff
@@ -72,6 +72,147 @@ set to the property is proposed to the user:
...
+<p align="center">
+  <img src="../images/preview_image.png" width="80%"/>
+</p>
+
+### Mock values
+
+The Markdown preview pane of Taipy Studio supports the binding of a property to a value.<br/>
+
+Because the Visual Studio code preview area is not connected to an actual application that could
+provide values set to variables, Taipy Studio uses a TODO taipy.mock.json
```
Comment (dinhlongviolin1) [inline docs/manuals/studio/gui.md]: Remove TODO

File: docs/manuals/studio/gui.md [hunk 3]
```diff
@@ -72,6 +72,147 @@ set to the property is proposed to the user:
...
+
+<p align="center">
+  <img src="../images/preview_data.png" width="80%"/>
+</p>
+
+!!! warning "No expression evaluation"
+    Expressions are not evaluated: you can use "{value}" to evaluate the variable in the preview
+    area, but expressions (such as "{value+1}") will **not** be replaced with their evaluation.
```
Comment (dinhlongviolin1) [inline docs/manuals/studio/gui.md:151]: You can set value+1 in mock file --> it can be replaced but not evaluated

**Review by dinhlongviolin1 — APPROVED** [review]
LGTM

File: docs/manuals/studio/gui.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/gui.md]: Right. That's how Microsoft documents it though.
But I get it. How do YOU do that on your Mac?

File: docs/manuals/studio/gui.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/gui.md]: 🙄 🙏

File: docs/manuals/studio/gui.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/gui.md:151]: Oh. I did not know that.

Comment (FabienLelaquais) [reply docs/manuals/studio/gui.md:151]: Actually @dinhlongviolin1 this does not seem to work.
I tried:

`file.md`:
```
<|{x}|slider|>

Double: <|{x*2}|input|>
```
with

`taipy.mock.json`:
```
{
  "x": 12,
  "x*2": 24
}
```

and the preview still shows an input control with "{x*2}" in it...

**Pull request merged**